.github/ALAppExtensions.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id></id>
```

Comment:
Consider making it clearer that this file is a template and not the actual nuspec. I can see there is a script to fill the missing fields, but looking only at this one is a bit confusing.

Maybe call it ALAppExtensions.template.nuspec? Or add a comment at the top?

Reply:
Renamed to ALAppExtensions.template.nuspec